maven/tp/Maven.target

```diff
@@ -108,6 +108,13 @@ SPDX-License-Identifier: EPL-2.0
           <version>32.0.1-jre</version>
           <type>jar</type>
         </dependency>
+        <dependencies>
```

Comment:
am not sure if this is sufficient or some category.xml or the like needs to be adjusted too